ruled_labels/[DRAFT]specs_monorepo.yaml

```diff
@@ -0,0 +1,187 @@
+
+name: Monorepo labels
+description: Import on 2023-05-09
+
+codes:
+  - code: A
+    name: Action 
+    description: used with GHA and trigger a certain process
+  - code: B
+    name: Dependencies 
+    description: Upgrade dependencies
```

Comment:
When reading the `B` labels, they are more about communicating breaking changes.  
But I cant think of a good title, but is definitely relevant for downstream Para* teams.